src/Xpress.jl

```diff
@@ -3,18 +3,20 @@ __precompile__()
 module Xpress
 
     using Libdl
-    const libxprs = string(Sys.iswindows() ? "" : "lib", "xprs", ".", Libdl.dlext)
 
     # Load in `deps.jl`, complaining if it does not exist
     const depsjl_path = joinpath(@__DIR__, "..", "deps", "deps.jl")
+
     if isfile(depsjl_path)
         include(depsjl_path)
     elseif !haskey(ENV, "XPRESS_JL_NO_DEPS_ERROR")
         error("XPRESS cannot be loaded. Please run Pkg.build(\"Xpress\").")
     else
-        const dlpath = ""
+        const xpressdlpath = ""
     end
 
+    const libxprs = joinpath(xpressdlpath, string(Sys.iswindows() ? "" : "lib", "xprs", ".", Libdl.dlext))
```

Comment:
do we need xpressdlpath here?

Reply:
I think it need it. It will be set to an empty string if `XPRESS_JL_NO_DEPS_ERROR` is set though, for when users want to load a library that is in their environment without using deps.jl.